cts/cts-support.in

```diff
@@ -23,6 +23,7 @@ CRM_EX_ERROR=1
 CRM_EX_USAGE=64
 
 UNIT_DIR="@systemdsystemunitdir@"
+RUNTIME_UNIT_DIR="/run/systemd/system"
```

Comment:
You can use @runstatedir@ instead of /run -- that way it handles older systems that use /var/run

(It's a recent change that configure.ac ensures runstatedir is defined, even if automake doesn't natively support it.)

Reply:
Done.